frontend/README.md

```diff
@@ -0,0 +1,117 @@
+# Frontend
+
+
+## Structure
+
+```bash
+# One folder per (micro)frontend.
+{service-name}/
+    # Compiled result of yarn build is stored in this folder
+    dist/
+    # Modules
+    node_modules/
+    # These files will be public, but other public files are added by webpack
+    # Normally, you will not have to add files here
+    public/
+        # Index file, modified by webpack during compilation to include the compiled sources in /src
+        index.html
+    src/
+        # Images, fonts, and other assets only used by this service.
+        assets/
+        # Contains all the service specific vue components, optionally grouped in folders
+        components/
+        # Root vue component that is mounted to the #app element by main.ts
+        App.vue
+        # The root file that mounts the App component to the DOM
+        main.ts
+        # Add syntax support in your IDE to write JSX-style typescript code (not used)
+        shims-tsx.d.ts
+        # Makes sure your IDE understands .vue files
+        shims-vue.d.ts
+    # Defines the browsers we support, e.g. used for css prefixes
+    .browserlistrc
+    # ESLint config
+    .eslintrc.js
+    # Dependency manager config (yarn is used)
+    package.json
+    # Vue cli configuration file (best to adjust webpack config here)
+    vue.config.js
+    # Versions of each dependency that was installed by yarn
+    yarn.lock
+
+# All shared scripts and assets
+shared/
+    # Fonts, images, icons used by multiple services
+    assets/
+    # Vue components used by multiple services, optionally grouped in folders
+    components/
+    # Styling and variables that is used by multiple services
+    scss/
+        base/
+            # Adds extra fonts
+            fonts.scss
+            # Reset all browser specific styles that we don't want to use
+            reset.scss
+            # Styling for text (use with @extend .style-name)
+            text-styles.scss
+            # Color and other variable definitions
+            variables.scss
+        # Style one or multiple elements that are reused often (buttons, inputs)
+        components/
+        # Styling single elements
+        elements/
+        # Define the layout of elements without defining the style of the elements
+        layout/
+    tsconfig.json
+```
+
+## Local development
+
+Inside a services directory, run `yarn serve` to start a development server with hot reloading. Use `yarn build` to compile to the `dist/` folder.
+
+*Hot reloading sometimes stops working in Safari and iOS. Restarting the server, clearing caches and closing the tab might solve the problems.*
```

Comment:
Any idea why?

Reply:
Not yet, use Firefox most of the time so I'm not going to look into this until I need to